src/reducers/books.js

```diff
@@ -1,21 +1,27 @@
 import { v4 as uuidv4 } from 'uuid';
+import { createSlice } from '@reduxjs/toolkit';
```

Comment:
- [ ] It would be better if we follow the guidelines by using `redux` instead of the `redux/toolkit`.